### PR TITLE
5.x.x/filter request tests

### PR DESCRIPTION
**Pull request by Pjiesco**

### CODE REVIEW

File: src/main/java/io/obswebsocket/community/client/message/request/filters/CreateSourceFilterRequest.java [hunk 1]
```diff
@@ -13,16 +13,16 @@
     private final Data requestData;
 
     @Builder
-    private CreateSourceFilterRequest(String sourceName, String filterName, Integer filterIndex, String filterKind, JsonObject filterSettings) {
+    private CreateSourceFilterRequest(String sourceName, String filterName, Boolean filterEnabled, Integer filterIndex, String filterKind, JsonObject filterSettings) {
```
Comment (ChristopheCVB) [inline src/main/java/io/obswebsocket/community/client/message/request/filters/CreateSourceFilterRequest.java:16]: I don't see any filterEnabled param for this request in the Google Sheet

File: src/test/java/io/obswebsocket/community/client/test/translator/requestSerializationTests/FilterRequestsSerializationTest.java [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+import org.json.JSONException;
+import org.junit.jupiter.api.Test;
+import org.skyscreamer.jsonassert.JSONAssert;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Fail.fail;
+
+public class FilterRequestsSerializationTest extends AbstractSerializationTest {
+    @Test
+    void createInputRequest() {
```
Comment (ChristopheCVB) [inline src/test/java/io/obswebsocket/community/client/test/translator/requestSerializationTests/FilterRequestsSerializationTest.java:19]: Wrong name?

**Review by ChristopheCVB — CHANGES_REQUESTED** [review]

File: src/main/java/io/obswebsocket/community/client/message/request/filters/CreateSourceFilterRequest.java (hunk 1 above)
Comment (Pjiesco) [reply src/main/java/io/obswebsocket/community/client/message/request/filters/CreateSourceFilterRequest.java:16]: oh, then i will  remove it. it was in the Data object so i just added it to the constructor

File: src/test/java/io/obswebsocket/community/client/test/translator/requestSerializationTests/FilterRequestsSerializationTest.java (hunk 2 above)
Comment (Pjiesco) [reply src/test/java/io/obswebsocket/community/client/test/translator/requestSerializationTests/FilterRequestsSerializationTest.java:19]: 🤦‍♂️yup

**Review by ChristopheCVB — APPROVED** [review]

**Pull request merged**